junifer/api/tests/test_functions.py

```diff
@@ -128,18 +147,30 @@ def test_run_single_element_with_preprocessing(
                 "kind": "fMRIPrepConfoundRemover",
             }
         ],
-        elements=[("sub-01",)],
+        elements=["sub-01"],
     )
     # Check files
     files = list(tmp_path.glob("*.sqlite"))
     assert len(files) == 1
 
 
+@pytest.mark.parametrize(
```

Comment:
Can you add a test case for the case we discussed?

Ej, sub-01 exists, but ses-02 does not.